VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -245,7 +309,10 @@ def get_total_used_size(self):
                 Utils.HandlerUtil.HandlerUtility.add_to_telemetery_data("ramDisksSize",str(total_used_ram_disks))
             if total_used_loop_device != 0 :
                 Utils.HandlerUtil.HandlerUtility.add_to_telemetery_data("loopDevicesSize",str(total_used_loop_device))
-            self.logger.log("Total used space in Bytes : {0}".format(total_used * 1024),True)
+            self.logger.log("TotalUsedSpaceAfterExcludeLUN in Bytes : {0}".format(total_used * 1024),True)
+            self.logger.log("TotalLUNExcludedUsedSpace in Bytes : {0}".format(excluded_disks_used *1024),True)
```

Comment:
Can this information be logged in same format as the one used by windows extension?